src/workerd/server/server.c++

```diff
@@ -3757,6 +3761,10 @@ class Server::WorkerLoaderNamespace: public kj::Refcounted {
             jsg::Lock& js, const Worker::Api& api, v8::Local<v8::Object> target) {
           env.populateJsObject(js, jsg::JsObject(target));
         },
+
+        .maybeOwnedSourceCode = !source.ownContentIsRpcResponse
+            ? kj::Maybe<kj::Own<void>>(kj::mv(source.ownContent))
+            : kj::Maybe<kj::Own<void>>(kj::none),
```

Comment:
@jasnell This case, if it ever happened, would lead to a UAF -- the RPC response would presumably be destroyed after the worker is loaded.

However, in workerd, `ownContentIsRpcResponse` is always false, so this won't happen.

But we should change this to either:
* Assert that it's false.
* Ignore the flag and always keep the owned content. This would be fine in workerd given it is single-threaded, so the response wouldn't be carried across event loops.